sdk/core/System.ClientModel/api/System.ClientModel.net6.0.cs

```diff
@@ -41,6 +41,22 @@ public partial class ClientResult<T> : System.ClientModel.ClientResult
         public static implicit operator T (System.ClientModel.ClientResult<T> result) { throw null; }
     }
 }
+namespace System.ClientModel.Options
+{
+    public partial class LoggingOptions
+    {
+        public LoggingOptions() { }
+        public bool IsLoggingContentEnabled { get { throw null; } set { } }
```

Comment:
It would be good to provide a sample showing what it looks like to turn off logging in a third-party client.